From 896dc6aee38898fe1c6822ee6a4a4989d21c650b Mon Sep 17 00:00:00 2001
From: Heiko Thiery <heiko.thiery@gmail.com>
Date: Thu, 25 Nov 2021 12:18:06 +0100
Subject: [PATCH] sysrepo BUGFIX build with musl libc fails due to unknown type
 name 'mode_t'

When building against the musl libc the build will fail due to the
missing sys/type.h include.

Signed-off-by: Heiko Thiery <heiko.thiery@gmail.com>
---
 src/plugins_datastore.h    | 1 +
 src/plugins_notification.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/plugins_datastore.h b/src/plugins_datastore.h
index a64e4131..d577e929 100644
--- a/src/plugins_datastore.h
+++ b/src/plugins_datastore.h
@@ -18,6 +18,7 @@
 #define _SYSREPO_PLUGINS_DATASTORE_H
 
 #include <stdint.h>
+#include <sys/types.h>
 
 #include <libyang/libyang.h>
 
diff --git a/src/plugins_notification.h b/src/plugins_notification.h
index 2099db62..f410e8d9 100644
--- a/src/plugins_notification.h
+++ b/src/plugins_notification.h
@@ -18,6 +18,7 @@
 #define _SYSREPO_PLUGINS_NOTIFICATION_H
 
 #include <stdint.h>
+#include <sys/types.h>
 
 #include <libyang/libyang.h>
 
-- 
2.30.2

